src/main/java/com/javadeobfuscator/deobfuscator/utils/TransformerHelper.java

```diff
@@ -202,6 +229,22 @@ public static int countOccurencesOf(MethodNode methodNode, int opcode) {
         return i;
     }
 
+    public static Map<Integer, AtomicInteger> calcOpcodeOccurenceMap(MethodNode methodNode) {
+        Map<Integer, AtomicInteger> map = new HashMap<>();
+        for (AbstractInsnNode abstractInsnNode : methodNode.instructions) {
+            int opcode = abstractInsnNode.getOpcode();
+            if (opcode >= 0) {
```

Comment:
When is opcode less than 0? Why should this be a check?

Reply:
frames and labels have opcode -1

Reply:
Hmmm, a possible remedy is to use `Utils.isInstruction(AbstractInsnNode)` which returns true if the list node isn't a line number/frame/label.